mariadb/run.sh

```diff
@@ -40,10 +37,10 @@ done
 
 # Init logins
 echo "[INFO] Init/Update users"
-for (( i=0; i < "$LOGINS"; i++ )); do
-    USERNAME=$(jq --raw-output ".logins[$i].username" $CONFIG_PATH)
-    PASSWORD=$(jq --raw-output ".logins[$i].password" $CONFIG_PATH)
-    HOST=$(jq --raw-output ".logins[$i].host" $CONFIG_PATH)
+for login in $(bashio::config "logins|keys"); do
+    USERNAME=$(bashio::config "logins[${login}].username")
+    PASSWORD=$(bashio::config "logins[${login}].password")
+    HOST=$(bashio::config "logins[${login}].host")
 
     if mysql -e "SET PASSWORD FOR '$USERNAME'@'$HOST' = PASSWORD('$PASSWORD');" 2> /dev/null; then
         echo "[INFO] Update user $USERNAME"
```

Comment:
Debug statements need to be replaced by Bashio.

e.g., `bashio::log.info "Info message"`